console/src/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// HGE common types
```

Comment:
`.d.ts` files are for declarations for JavaScript modules. Please move this type to `types.ts` in the same file. 
Note for the feature: we should provide exhaustive typings for RQL queries and specify that `type` is a union of given string literals.